packages/@react-spectrum/cards/test/CardView.test.js

```diff
@@ -1064,31 +1064,29 @@ describe('CardView', function () {
     `('$Name CardView should call loadMore when scrolling to the bottom', function ({layout}) {
       let onLoadMore = jest.fn();
       let tree = render(<DynamicCardView layout={layout} onLoadMore={onLoadMore} />);
-      expect(tree).toBeTruthy();
 
-       // TODO: loadMore is being called twice on mount??? debug this
-      // act(() => {
-      //   jest.runAllTimers();
-      // });
+      act(() => {
+        jest.runAllTimers();
+      });
 
-      // let cards = tree.getAllByRole('gridcell');
-      // expect(cards).toBeTruthy();
-      // expect(onLoadMore).toHaveBeenCalledTimes(0);
-      // triggerPress(cards[1]);
+      let cards = tree.getAllByRole('gridcell');
+      expect(cards).toBeTruthy();
+      // Virtualizer calls onLoadMore twice due to initial layout
+      expect(onLoadMore).toHaveBeenCalledTimes(2);
+      triggerPress(cards[1]);
```

Comment:
Turns out Virtualizer calls `onLoadMore` several times on render for all Virtualizer components, adapting test to match the current behavior. Might be an idea to have the virtualizer.layout have a property that can let the Virtualizer know if it is still doing first time layout validation so we don't call `onLoadMore` early